scripts/sysman/fabric.yml

```diff
@@ -304,6 +304,26 @@ returns:
     - $X_RESULT_ERROR_INSUFFICIENT_PERMISSIONS:
         - "User does not have permissions to query this telemetry."
 --- #--------------------------------------------------------------------------
+type: function
+desc: "Get Fabric port throughput from multiple ports in a single call"
+class: $sFabricPort
+name: GetMultiPortThroughput
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: "$s_fabric_port_handle_t*"
+      name: phPort
+      desc: |
+            [in][range(0, *pCount)] array of handle of components of this type.
+            if count is less than the number of components of this type that are available, then the driver shall only retrieve that number of component handles.
+    - type: uint32_t
+      name: Count
+      desc: "[in] Number of ports enumerated"
+    - type: $s_fabric_port_throughput_t**
```

Comment:
Does sysman allocate the memory for these handles and throughputs? The `details:` section should specify the rules for the handles and throughputs memories. If the user is expected to pre-allocate and manage the memory, then the spec should call that out along with the expected lifetime of that memory as it pertains to sysman (for example, is the memory ever accessed by sysman after this function is called? If not, then the memory is entirely user-managed). If sysman is allocating the backing memory for these arrays, then it needs to free those resources, and the user needs a mechanism to inform it when to do so.

Reply:
Fixed as per your feedback